Packs/Kafka/Integrations/KafkaV3/README.md

```diff
@@ -27,7 +27,8 @@ This integration is fully compatible with the Kafka v2 integration.
     | Topic to fetch incidents from (Required for fetch incidents) |  | False |
     | CSV list of partitions to fetch messages from |  | False |
     | Offset to fetch messages from (exclusive) | The initial offset to start fetching from, not including the value set \(e.g., if 3 is set, the first event that will be fetched will be with offset 4\). If you want to start from the earliest or latest, type in 'earliest' or 'latest' accordingly. | False |
-    | Max number of messages to fetch |  | False |
+    | Maximum number of messages to fetch |  | False |
+    | Stop consuming upon timeout | When fetching a large amount of messages (100+), we recommend to terminate the consumption of messages upon timeout. This will lead to the termination of the fetch when no messages are received after a defined period of time, instead of continuing to request messages until the number of messages specified in the 'max_fetch' parameter is reached. | False |
```

Comment:
Lets change according to the yml

Reply:
Resolved - was changed to the suggestion.